packages/cosmic-swingset/test/export-storage.test.ts

```diff
@@ -1,29 +1,44 @@
-// @ts-check
-import test from 'ava';
+import type { TestFn } from 'ava';
+import anyTest from 'ava';
 import { exportStorage } from '../src/export-storage.js';
 
-const makeBatchChainStorage = published => {
-  const deleted = [];
-  const batchChainStorage = (method, args) => {
-    const lookup = path => {
-      let node = published;
-      if (path === 'published') return node;
+const test = anyTest as TestFn;
+
+type PublishedNode = {
+  _?: unknown;
+  [key: string]: PublishedNode | unknown;
+};
```

Comment:
I think this can be made to better reflect the design (prettify to taste):
```suggestion
type PublishedNode = { _?: unknown } & // data contents at this node
  { [key: string]: PublishedNode }; // subnodes
```